src/main/java/seedu/workbook/logic/commands/FindCommand.java

```diff
@@ -2,9 +2,11 @@
 
 import static java.util.Objects.requireNonNull;
 
+import java.util.function.Predicate;
+
 import seedu.workbook.commons.core.Messages;
 import seedu.workbook.model.Model;
-import seedu.workbook.model.internship.CompanyContainsKeywordsPredicate;
+import seedu.workbook.model.internship.Internship;
 
 /**
  * Finds and lists all internships in WorkBook whose company name contains
```

Comment:
Could you also update the javadocs here? Something to the effect of:
"Filters for internships either by Company, Role, or Stage name as indicated by prefix."